### PR TITLE
fixing urls and names

### PR DESCRIPTION
**Pull request by zekth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +121,6 @@ Consider submitting to [the deno.land/x](https://github.com/denoland/registry/bl
 
 ## Chinese
 
-- [deno.js.cn](https://deno.js.cn/)
```
Comment (bartlomieju) [inline README.md]: No longer working?

Comment (zekth) [reply README.md]: nop. I just found it's registered by @justjavac . Is there a problem?

**Comment by zekth** [comment]
No answer. Landing it.

**Comment by kevinkassimo** [comment]
@zekth I remember @justjavac mentioned about some issue with the domain name. Let's drop it for now.

**Pull request merged**